CRM/Core/BAO/File.php

```diff
@@ -114,7 +114,7 @@ public static function filePostProcess(
 
     $config = CRM_Core_Config::singleton();
 
-    $path = explode('/', $data);
+    $path = explode(DIRECTORY_SEPARATOR, $data);
```

Comment:
Thanks for the PR but I don't think this is right because on windows you can have both, mixed within the same path. I normally "normalize" the input so it's all forward slashes and then just use forward slash, and then you just have to be careful about parameters to calls that are actually run by the OS, which rename() actually might be. It can be finicky so I was putting it off.

Reply:
@demeritcowboy ah ok - I couldn't see why it wasn't done cos it looked like the answer was there in the ticket